src/java/org/apache/solr/perf/Args.java

```diff
@@ -95,6 +95,10 @@ public long getLong(String argName) {
     return Long.parseLong(getString(argName));
   }
 
+  public boolean getBool(String argName) {
+    return Boolean.getBoolean(getString(argName));
```

Comment:
@vthacker I don't think Boolean.getBoolean is correct. It will lookup the system property given to its argument. I think you mean Boolean.parseBoolean instead?